shell/browser/atom_autofill_driver.cc

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) 2019 GitHub, Inc.
+// Use of this source code is governed by the MIT license that can be
+// found in the LICENSE file.
+
+#include "shell/browser/atom_autofill_driver.h"
+
+#include <utility>
+
+#include "content/public/browser/render_widget_host_view.h"
+#include "shell/browser/api/atom_api_web_contents.h"
+#include "shell/browser/native_window.h"
+
+namespace electron {
+
+AutofillDriver::AutofillDriver(content::RenderFrameHost* render_frame_host)
+    : render_frame_host_(render_frame_host), binding_(this) {
+  autofill_popup_.reset(new AutofillPopup());
+}
+
+AutofillDriver::~AutofillDriver() {}
+
+void AutofillDriver::BindRequest(
+    mojom::ElectronAutofillDriverAssociatedRequest request) {
+  binding_.Bind(std::move(request));

```

Comment:
Based on the lazy load suggestion, its simpler to bind to the request in the constructor rather than a specific method since the lifetime is always connected to one render frame host. If there are multiple requests per frame , then its better to have a separate method like this and also in that case existing connections should be unbound before creating a new one.